sdk/python/kfp_tekton/compiler/_op_to_template.py

```diff
@@ -256,7 +256,7 @@ def _process_parameters(processed_op: BaseOp,
                 if mount_path not in mounted_param_paths:
                     _add_mount_path(name, path, mount_path, volume_mount_step_template, volume_template, mounted_param_paths)
             # Record what artifacts are moved to result parameters.
-            parameter_name = sanitize_k8s_name(processed_op.name + '-' + name, allow_capital_underscore=True)
+            parameter_name = sanitize_k8s_name(processed_op.name + '-' + name, allow_capital_underscore=True, max_length=float('Inf'))
```

Comment:
I'm okay with not limit the length for artifacts. Just be aware that we still cannot have artifact name that is longer than the max length for the object storage path (e.g. S3's max length is 1024 chars)

Reply:
Sure. A validation on length of output_name can be made to avoid that. It doesn't tend to be nearly as big as that though, I think.